Makefile

```diff
@@ -32,8 +32,11 @@ export GOPROXY
 # Active module mode, as we use go modules to manage dependencies
 export GO111MODULE=on
 
+# Go version
+GOLANG_VERSION := 1.22.8
```

Comment:
1.22.11 use the latest as we already have in the dockerfile https://github.com/kubernetes-sigs/cluster-api-provider-gcp/blob/main/Dockerfile#L16

Reply:
oh i think capi is using 1.23 but we can do that in a follow up

Reply:
Updated to 1.22.11 thanks @cpanato

Reply:
@cpanato BTW it looks like CAPI 1.9.4 is still using go 1.22 https://github.com/kubernetes-sigs/cluster-api/blob/79e6731b0d20a0ca2dad5e4f96d2a532aa82b54b/Makefile#L26